pkg/dockerfile/dockerfile.go

```diff
@@ -111,9 +111,33 @@ func Parse(b []byte) ([]instructions.Stage, []instructions.ArgCommand, error) {
 	if err != nil {
 		return nil, nil, err
 	}
+
+	metaArgs = stripEnclosingDoubleQuotes(metaArgs)
+
 	return stages, metaArgs, nil
 }
 
+// stripEnclosingDoubleQuotes removes double quotes enclosing the value of each instructions.ArgCommand in a slice
+func stripEnclosingDoubleQuotes(metaArgs []instructions.ArgCommand) []instructions.ArgCommand {
```

Comment:
We can add table driven unit tests here
1. value like`"foo"`
2. For empty value `""` ? What happens then?
   maybe we shd keep quotes as is then?
3.  value `foo`
4.  invalid value `"foo` shd probably error out?

Reply:
Some other test cases to add would be
5. When `metaArgs` is empty
6. Multiple `mataArgs` instructions have `""`. (All get resolved)

Reply:
just something to consider, something like `\"foo\"` would probably need to maintain the quotes

Reply:
> We can add table driven unit tests here for this function only
> 
> 1. value like`"foo"`
> 2. For empty value `""` ? What happens then?
>    maybe we shd keep quotes as is then?
> 3. value `foo`
> 4. invalid value `"foo` shd probably error out?

2. Docker seems to strip out the quotes and treat it as a valid, but blank string
4. Yup, Docker also seems to error when either quote isn't matched

We also might want to handle single quotes as docker seems to handle those as well

Reply:
> just something to consider, something like `\"foo\"` would probably need to maintain the quotes

Ya, it seems that Docker does not remove the quotes in that case which will result in an error for things like image name